catroid/src/main/java/org/catrobat/catroid/merge/MergeTask.java

```diff
@@ -60,21 +54,11 @@
 	private Project firstProject = null;
```

Comment:
I still don't like that the values of these Scenes and Projects depend on which Constructor is used in the first place.
For example if i create an instance with `MergeTask(Scene firstScene, Scene secondScene, Context context)` and then call `mergeProjects(String name)` firstProject and secondProject will be null but used in mergeProjects.

there is no clear border between those scenarios